install/baltrad/fix_macro_beamb.patch

```diff
@@ -9,5 +9,5 @@ index 3caf9cd..49b79ea 100644
 -#define MOD_INIT_SETUP_TYPE(itype, otype) Py_TYPE(&itype) = otype
```

Comment:
@ghiggi Can we exclude the install-folder? We do not want to change anything there as we just fetch it from upstream.